dgraph/cmd/raft-migrate/run.go

```diff
@@ -50,13 +50,22 @@ func init() {
 
 	flag := RaftMigrate.Cmd.Flags()
 	flag.StringP("old-dir", "", "", "Path to the old (z)w directory.")
-	flag.IntP("old-node-id", "", 1,
-		"Node ID of the old node. This will be the node ID of the new node.")
-	flag.IntP("old-group-id", "", 0, "Group ID of the old node. This is used to open the old wal.")
 	flag.StringP("new-dir", "", "", "Path to the new (z)w directory.")
 	enc.RegisterFlags(flag)
 }
 
+func updateEntry(entry raftpb.Entry) raftpb.Entry {
+	// Raft commits an empty entry on becoming leader.
+	if entry.Type == raftpb.EntryConfChange || len(entry.Data) == 0 {
+		return entry
+	}
+
+	data := make([]byte, 8+len(entry.Data))
+	copy(data[8:], entry.Data)
```

Comment:
Don't you need to set the key on the first 8 bytes?

Reply:
Add a comment if we don't need the key

Reply:
Done